dataland-backend/src/main/kotlin/org/dataland/datalandbackend/entities/StoredCompanyEntity.kt

```diff
@@ -62,7 +65,7 @@ data class StoredCompanyEntity(
     @JsonValue
     override fun toApiModel(viewingUser: DatalandAuthentication?): StoredCompany {
         val identifierMap = mutableMapOf<IdentifierType, MutableList<String>>()
-        for (identifierType in IdentifierType.values()) {
+        for (identifierType in IdentifierType.entries) {
```

Comment:
Why entries instead of values()? (same comment as below)

Reply:
entries does the same as values(), probably just the newer version. therefore i will close this conversation